_posts/2022-01-30-my-journey-in-openfaas-community.md

```diff
@@ -0,0 +1,64 @@
+---
+title: My Journey in OpenFaaS Community So Far
+description: "Nitishkumar shares his experience of working in the OpenFaaS community"
```

Comment:
"Nitishkumar shares his experiences from contributing code to OpenFaaS and joining the community"

Reply:
Thank You for suggestion. Changes are done.